### PR TITLE
talks admin create, edit, delete now working.

### PR DESCRIPTION
**Pull request by sbatschelet**
* add admin/talks/create, edit and delete routes.
* add bootstrap-datetimepicker plugin to handle date/time form fields.
* add engine JSON allow_blessed to main config to support passing DT object to $form->fill

### CODE REVIEW

File: lib/PerlDance/Routes/Admin.pm [hunk 1]
```diff
@@ -162,4 +166,132 @@ get '/admin/talks' => require_role admin => sub {
...
+            duration        => $talk->duration,
+            title           => $talk->title,
+            tags            => $talk->tags,
+            abstract        => $talk->abstract,
+            url             => $talk->url,
+            comments        => $talk->comments,
+            accepted        => $talk->accepted,
+            confirmed       => $talk->confirmed,
+            lightning       => $talk->lightning,
+            start_time      => schema->format_datetime($talk->start_time),
```
Comment (SysPete) [inline lib/PerlDance/Routes/Admin.pm]: Should pass DateTime to view and handle formatting/display there via filter or widgets.

Comment (sbatschelet) [inline lib/PerlDance/Routes/Admin.pm]: @SysPete yeah I will continue work adding datetimepicker feature https://github.com/Eonasdan/bootstrap-datetimepicker

**Comment by sbatschelet** [comment]
@SysPete I think this is an improvement.  admin/talks/create the DT is defined by bootstrap-datetimepicker plugin.  With admin/talks/edit the plugin is set by talk->start_date DT directly then can be adjusted in the view as needed.

**Comment by racke** [comment]
@SysPete: ready to merge this?

**Comment by SysPete** [comment]
@racke discussed the JSON allow_blessed with @sbatschelet yesterday and he will remove that + make other changes to handle that being removed. Once those changes are in then we're ready for merge.

**Comment by sbatschelet** [comment]
@racke I will complete today.

**Pull request closed without merging**